components/providers.js

```diff
@@ -239,6 +213,148 @@ const DomainURL = styled.div`
   color: #a5a5a5;
 `;
 
+const PROVIDERS = [
+  {
+    name: 'ZEBEDEE',
+    image: '/images/zbd.svg',
+    imageStyle: { width: '130px', marginTop: '10px', marginBottom: '3px' },
+    lightningAddressDomain: 'zbd.gg',
+    url: 'https://zebedee.io/app',
+    buttonText: 'Download App',
+  },
+  {
+    name: 'BTCPay Server',
+    image: '/images/btcpay.svg',
+    imageStyle: { width: '95px' },
+    lightningAddressDomain: 'yourbtcpay.server',
+    url: 'https://btcpayserver.org/',
+  },
+  {
+    name: 'Bitrefill',
+    image: '/images/bitrefill.svg',
+    imageStyle: { width: '125px' },
+    lightningAddressDomain: 'bitrefill.me',
+    url: 'https://bitrefill.com/',
+  },
+  {
+    name: 'LNTXBot',
+    image: '/images/lntxbot.png',
+    imageStyle: { width: '130px', marginBottom: '-5px' },
+    lightningAddressDomain: 'lntxbot.com',
+    url: 'http://lntxbot.fiatjaf.com/',
```

Comment:
@fiatjaf just FYI, I noticed this URL is broken. Not sure if you'd like it swapped out for something else (like [this](https://fiatjaf.com/2495e7ef.html) or [this](https://telegram.me/lntxbot)).

Reply:
Oh, please swap it for https://lntxbot.com if you can, please!

Reply:
will do ⚡

Reply:
made it uniform across the site so should be good now